source/documentation/Get started/quickstart.md

```diff
@@ -0,0 +1,210 @@
+- [MoJ Analytical IT Tools Strategy](https://moj-analytical-services.github.io/moj-analytical-it-tools-strategy/): describes recommended ways of working on the Analytical Platform
+
+## 2. Create Slack account
+
+We use Slack to communicate status updates, such as scheduled maintenance for the Analytical Platform. You can also use it to communicate with our support team and the Analytical Platform user community. 
+
+There are two workspaces we recommend joining: [Justice Digital](https://mojdt.slack.com/) and [ASD](https://asdslack.slack.com/). To join, while signed in to your work email, navigate to each workspace and request to join. Note that workspace moderators only consider users from the following email addresses:
+
+- @justice.gsi.gov.uk
+- @digital.justice.gov.uk
+- @cjs.gsi.gov.uk
+- @noms.gsi.gov.uk
+- @legalaid.gsi.gov.uk
+- @justice.gov.uk
+- @judiciary.uk
+
+> **Note**: You can only access the Justice Digital and ASD workspaces using DOM1 and MoJ Digital and Technology MacBooks. If you use Quantum, you will need to access Slack using a mobile device instead.
+
+### Join Slack channels
+
+Join the following Slack channels in the **ASD workspace**:
+- **#ask-data-engineering**: for discussing data engineering and making technical queries to the Data Engineering team regarding Airflow
+- **#data_science**: for discussing data science tools and techniques with the Ministry of Justice's Data Science community
+- **#git**: for discussing Git tooling with the wider Ministry of Justice community
+- **#python**: for discussing Python programming with the wider Ministry of Justice community
+- **#r** and **#intro_r**: for discussing R programming with the wider Ministry of Justice community; #intro_r is aimed at new users
+
+Additionally, in the Justice Digital **workspace** join the following:
+- [**#analytical-platform-support**](https://mojdt.slack.com/archives/C4PF7QAJZ): for tracking [support queries raised by users of the Analytical Platform on GitHub Issues](https://github.com/ministryofjustice/data-platform-support/issues)
+- [**#ask-operations-engineering**](https://mojdt.slack.com/archives/C01BUKJSZD4): for requesting support with GitHub; you can use this channel to request access to the Analytical Platform later in this guide
+
+## 3. Create GitHub account
+
+Using your work email address (ending either **justice.gov.uk** or **digital.justice.gov.uk**), [sign up for a GitHub account](https://github.com/join). See the [GitHub documentation](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account) for instructions. Ensure that:
+
+- your account uses the free plan subscription
+- you choose a username that does not contain upper-case characters (for good practice)
+- you set your Git username for **every** repository on your device; see the [GitHub documentation](https://docs.github.com/en/get-started/getting-started-with-git/setting-your-username-in-git) for instructions
```

Comment:
I think this would be unclear for people unfamiliar with git

```suggestion
- you set your username as the 'global' default on your device (so all of your git commands are run as you); see the [GitHub documentation](https://docs.github.com/en/get-started/getting-started-with-git/setting-your-username-in-git) for instructions
```